connect/client.go

```diff
@@ -106,6 +109,53 @@ type restClient struct {
 	client    httpClient
 }
 
+func (rs *restClient) GetFiles(itemUUID string, vaultUUID string) ([]onepassword.File, error) {
+	span := rs.tracer.StartSpan("GetFiles")
+	defer span.Finish()
+
+	jsonURL := fmt.Sprintf("/v1/vaults/%s/items/%s/files", vaultUUID, itemUUID)
```

Comment:
I think this is not only true for these specific functions, but these parameters could use some validation. My guess is that passing an empty value for either of them will lead to a weird error from Connect because we're hitting a different route.

Reply:
I think you are right, I am going to open a different PR and validate the paths within the `buildRequest` method.